core/rawdb/accessors_skipped_txs.go

```diff
@@ -157,6 +157,24 @@ func WriteSkippedTransaction(db ethdb.Database, tx *types.Transaction, traces *t
 	}
 }
 
+func ResetSkippedTransactionTracesByHash(db ethdb.Database, txHash common.Hash) {
+	mu.Lock()
+	defer mu.Unlock()
+
+	data := readSkippedTransactionRLP(db, txHash)
+	if len(data) == 0 {
+		return
+	}
+	var stx SkippedTransaction
+	if err := rlp.Decode(bytes.NewReader(data), &stx); err != nil {
+		log.Crit("Invalid skipped transaction RLP", "hash", txHash.String(), "data", data, "err", err)
+	}
+	if stx.BlockHash != nil && *stx.BlockHash == (common.Hash{}) {
+		stx.BlockHash = nil
+	}
```

Comment:
```suggestion
	stx := ReadSkippedTransaction(db, txHash)
```

Reply:
fixed in https://github.com/scroll-tech/go-ethereum/pull/501/commits/1b7db0024f6f413d9958c1cfcbf281abdf1db45b